JSTOR.js

```diff
@@ -246,6 +246,19 @@ function processRIS(text, jid) {
 		item.url = item.url.replace('http:', 'https:'); // RIS still lists http addresses while JSTOR's stable URLs use https
 		if (item.url && !item.url.startsWith("http")) item.url = "https://" + item.url;
 		
+		// remove all caps from titles and authors.
+		for (i = 0; i < item.creators.length; i++) {
+			if (item.creators[i].lastName && item.creators[i].lastName == item.creators[i].lastName.toUpperCase()) {
+				item.creators[i].lastName = ZU.capitalizeTitle(item.creators[i].lastName.toLowerCase(), true);
```

Comment:
Any reason for this instead of `capitalizeName()`?

Reply:
I don't know `capitalizeName()` -- do you have a link to definition or code? I couldn't find either a generic js function nor something in Zotero

Reply:
Zotero.Utilities functions are all in a separate repo now:

https://github.com/zotero/utilities/blob/48d5e170545acaa9127612d6d4337c032864b564/utilities.js#L121-L141